### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by dmarcoux**
Following our retrospective discussions, we agreed to have a pull request template to promote good practices in pull requests we create and receive. This is for us, but also for our contributors.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+WIP
+
+Includes:
+- Explain what we expect (links to CONTRIBUTING.md)
+- Screenshots, GIFs, review app (visual changes)
```
Comment (saraycp) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would explain somewhere how to add review app. Perhaps simply adding to this line "just add the label 'review up' to the PR" or something.

LGTM anyway

Comment (dmarcoux) [reply .github/PULL_REQUEST_TEMPLATE.md]: I will write more than this. It's just that it's Friday afternoon, so this is more like a reminder :slightly_smiling_face:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+WIP
```
Comment (Ana06) [inline .github/PULL_REQUEST_TEMPLATE.md]: :eyes:

Comment (Ana06) [inline .github/PULL_REQUEST_TEMPLATE.md]: :eyes:

**Comment by dmarcoux** [comment]
@openSUSE/obs-frontend Please review. I went with something short, straight to the point.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+If you haven't done so already, please read the
+[CONTRIBUTING.md](https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md)
+to learn how we work and what we expect from all contributors.
+
+In order to help reviewers, it is always good have some context about your
```
Comment (vpereira) [inline .github/PULL_REQUEST_TEMPLATE.md]: I liked it. I'm just not sure if this paragraph shouldn't be in the [CONTRIBUTING.md](https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md) and in the PR template just remember that before to submit a PR, the contributor should actually read the ```CONTRIBUTING.md``` and make sure that tests are passing before open a PR (unless it's a WIP) ?

Comment (dmarcoux) [reply .github/PULL_REQUEST_TEMPLATE.md]: For the tests, it's already addressed in the [CONTRIBUTING.md](https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md#mind-the-automated-reviews).

As for this paragraph, I will move it to the CONTRIBUTING.md file.

**Pull request merged**